modules/oc-mirror-operator-catalog-filtering.adoc

```diff
@@ -0,0 +1,271 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-operator-catalog-filtering_{context}"]
+= Operator catalog filtering
+
+The list of bundles selected for mirroring by oc-mirror v2 is based on processing the catalog contents based on the information in `imageSetConfig`.
+
+During the selection of bundles for mirroring, oc-mirror v2 does not seek to bridge the gap graph between two channels or catalogs. Instead, it rigorously adheres to the user's instructions outlined by the `imageSetConfig`.
+
+When oc-mirror v2 selects bundles for mirroring, it refrains from inferring Group Version Kind (GVK) or bundle dependencies, omitting them from the mirroring set. It strictly adheres to the user's instructions. Therefore, you must explicitly specify any required dependent packages and their versions.
+
+Most bundle versions adhere to semantic versioning standards. Bundles within a channel can be sorted by version, allowing selection of bundles falling within the specified range in the `ImageSetConfig`.
+
+Such an algorithm will lead to consistent results compared to v1 results. Nevertheless, the upgrade graph information conveyed by the `replaces`, `skip` and `skipRange` is not taken into account. This algorithm is totally different from the one used by OLM. There is no guarantee here that the selected list of bundles would allow OLM to perform an upgrade for the selected operator. There may also be a much shorter path between the minVersion and the maxVersion requested. oc-mirror will, in this case, mirror more bundles than what a cluster would need to perform the upgrade.
+
+
+.Scenarios:
+[cols="1,2",options="header"]
+
+|===
+
+|ImageSetConfig operator filtering 
+|Expected bundle versions
+
+
+a|Scenario 1
+
+[source,yaml]
+----
+mirror:
+ operators:
+   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.10
```

Comment:
can we change the catalog in this complete section to a much later version like v4.15